atlas/packages/app/src/shared/components/Grid/Grid.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import useResizeObserver from 'use-resize-observer'
+
+const Container = styled.div<GridProps>`
+  display: grid;
+  gap: ${(props) => props.gap};
+  grid-template-columns: repeat(${(props) => `auto-${props.repeat}`}, minmax(min(270px, 100%), 1fr));
+`
+
+type GridProps = {
+  gap?: number | string
+  repeat?: 'fit' | 'fill'
+  className?: string
+  onResize?: (sizes: number[]) => void
+}
+const Grid: React.FC<GridProps> = ({ children, className, repeat = 'fit', onResize, ...props }) => {
```

Comment:
Shouldn't `gap` have some default value here or in the `Container`?

Reply:
Good idea, i'll add `spacing.xs` as a default value